targets/ice40_up5k_b_evn/base.py

```diff
@@ -122,8 +117,7 @@ def __init__(self, platform, **kwargs):
 
         # Disable final deep-sleep power down so firmware words are loaded
         # onto softcore's address bus.
-        platform.toolchain.build_template[3] = "icepack -s {build_name}.txt {build_name}.bin"
-        platform.toolchain.nextpnr_build_template[2] = "icepack -s {build_name}.txt {build_name}.bin"
+        platform.toolchain.build_template[2] = "icepack -s {build_name}.txt {build_name}.bin"
```

Comment:
This should call `platform_toolchain_extend` I think?

Reply:
`-s` is now passed to `icepack` in LiteX by default, so this update is not needed anymore